### PR TITLE
ID#11430 - Oracle Access Governance with OCI IAM - Changes for the July release of AG

### PR DESCRIPTION
**Pull request by anbuoracle**
ID#11430 - Oracle Access Governance with OCI IAM 
Changes for the July release of AG

# Add here your WMS ID

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

File: accessgovernance-oci-iam/workshops/freetier/manifest.json [hunk 1]
```diff
@@ -23,13 +23,13 @@
...
-            "title": "Lab 3: Setup and configure Oracle Access Governance service instance",
-            "description": "Setup and configure Oracle Access Governance service instance ",
-            "filename": "../../instance-setup/ag-instance-setup.md"
+            "title": "Lab 3: Create OCI Policies, Groups, Compartments and OCI IAM Users ",
+            "filename": "../../create-oci-resource/create-oci-resource.md"
         },
         {
-            "title": "Lab 4: Create OCI Policies, Groups and Compartments ",
-            "filename": "../../create-oci-resource/create-oci-resource.md"
+            "title": "Lab 4: Setup and configure Oracle Access Governance service instance",
```
Comment (aberinnj) [inline accessgovernance-oci-iam/workshops/freetier/manifest.json:30]: Noticing that these Labs do not match what is in your Lab's outline on WMS, please update WMS accordingly

**Review by aberinnj — CHANGES_REQUESTED** [review]

**Comment by aberinnj** [comment]
Checked live version, lgtm

**Comment by anbuoracle** [comment]
Hi Norman

I’ll update WMS as soon as the pull request is approved as I don’t want to change the outline in production before the code gets in.


Thanks
Anbu

From: Norman Aberin ***@***.***>
Date: Tuesday, July 11, 2023 at 1:07 PM
To: oracle-livelabs/security ***@***.***>
Cc: Anbu Anbarasu ***@***.***>, Author ***@***.***>
Subject: [External] : Re: [oracle-livelabs/security] ID#11430 - Oracle Access Governance with OCI IAM - Changes for the July release of AG (PR #107)

@aberinnj requested changes on this pull request.

________________________________

In accessgovernance-oci-iam/workshops/freetier/manifest.json<https://urldefense.com/v3/__https:/github.com/oracle-livelabs/security/pull/107*discussion_r1260099898__;Iw!!ACWV5N9M2RV99hQ!O2PgGkJGUnbmdjOtckyEMJbWAXnr3mgKvEnK6VWvkNErT0O3xIJ61ivdD8MtvXQZKQHcXHtn745_Z_TKs0l6dXy-7RQ$>:

>          },

         {

-            "title": "Lab 4: Create OCI Policies, Groups and Compartments ",

-            "filename": "../../create-oci-resource/create-oci-resource.md"

+            "title": "Lab 4: Setup and configure Oracle Access Governance service instance",

Noticing that these Labs do not match what is in your Lab's outline on WMS, please update WMS accordingly

File: accessgovernance-oci-iam/workshops/freetier/manifest.json (hunk 1 above)
Comment (anbuoracle) [reply accessgovernance-oci-iam/workshops/freetier/manifest.json:30]: updated WMS

**Review by aberinnj — APPROVED** [review]

**Pull request merged**